packages/extension-sample/src/setup-server.ts

```diff
@@ -7,16 +7,27 @@
 
 import {
     Application as ExpressApplication,
-    ApplicationExtension as ExpressApplicationExtension
+    ApplicationExtension as ExpressApplicationExtension,
+    ApplicationExtensionConfig
 } from '@salesforce/pwa-kit-runtime/ssr/server/extensibility'
 
-class SampleExtension extends ExpressApplicationExtension {
+interface Config extends ApplicationExtensionConfig {
+    path?: string
+
+}
```

Comment:
Hmmm I see that because we have react app extension and express app extension sandwiched into one project it becomes a problem. 

Maybe in `src/types` we can have files for both `config-server` and `config-react` then import those into the setup-server setup-app? 

I think this is bringing up the question of would having a separate application extension project be something that would help solve problems like this 🤔

Reply:
I'll define the type for just a single shared config. Let's see if developers really need the distinct configurations for server/react or not.